types/sse.d.ts

```diff
@@ -0,0 +1,90 @@
+export type SSE = {
+    /**
```

Comment:
constructor is missing here
```suggestion
    new (url: string, options?: SSEOptions): SSE;
    /**
```